Source/DataSourceProvider.swift

```diff
@@ -38,7 +38,9 @@ where CellFactory.Item == DataSource.Item, SupplementaryFactory.Item == DataSour
 
     fileprivate var bridgedDataSource: BridgedDataSource?
 
-
+    //The data source that provides the editing functionality on the tableView
+    public var dataSourceTableEditing: DataSourceTableEditingProtocol?
```

Comment:
it would be nice to *only* have this property for `tableView` dataSources... but I'm not sure if we can express that?

perhaps:

1. this could be `private` (or `fileprivate`)
2. then, we could expose the setter for *only* `tableView` dataSources

?